proto/buf/alpha/registry/v1alpha1/plugin_curation.proto

```diff
@@ -381,6 +381,8 @@ message CreateCuratedPluginRequest {
   string license_url = 16;
   // Visibility indicates whether the plugin is public or private.
   CuratedPluginVisibility visibility = 17;
+  // The URL leading to the integration guide of the plugin, if available.
+  string integration_guide_url = 18;
```

Comment:
Shouldn't this also be added to `CuratedPlugin` message type?

Reply:
Sure, I can add it in the same PR. I wanted to break this into 2 PR's, the config+request side, but no issue also updating the core type.